modules/140-user-authz/docs/USAGE.md

```diff
@@ -29,102 +29,112 @@ spec:
 ## Creating a user
 
 There are two types of users in Kubernetes:
+
 * Service accounts managed by Kubernetes via the API;
 * Regular users managed by some external tool that the cluster administrator configures. There are many authentication mechanisms and, accordingly, many ways to create users. Currently, two authentication methods are supported:
-    * Via the [user-authn](../../modules/150-user-authn/) module.
-    * Via the certificates.
+  * Via the [user-authn](../../modules/150-user-authn/) module.
+  * Via the certificates.
 
 When issuing the authentication certificate, you need to specify the name (`CN=<name>`), the required number of groups (`O=<group>`), and sign it using the root CA of the cluster. It is this mechanism that authenticates you in the cluster when, for example, you use kubectl on a bastion node.
 
-### Creating a ServiceAccount and granting it access
-* Create a `ServiceAccount` in the `d8-service-accounts` namespace
+### Creating a ServiceAccount for a machine and granting it access
 
-  An example of creating `gitlab-runner-deploy` `ServiceAccount`:
-  ```bash
-  kubectl -n d8-service-accounts create serviceaccount gitlab-runner-deploy
-  ```
+It may be required to give your machine static access to the Kubernetes API, e.g., for deploying applications from a CI system runner.
 
-* Grant the necessary rights to the `ServiceAccount` (using the [ClusterAuthorizationRule](cr.html#clusterauthorizationrule) CR)
+1. Create a `ServiceAccount` in the `d8-service-accounts` namespace (you can change the name):
 
-  An example:
-  ```bash
-  kubectl create -f - <<EOF
-  apiVersion: deckhouse.io/v1
-  kind: ClusterAuthorizationRule
-  metadata:
-    name: gitlab-runner-deploy
-  spec:
-    subjects:
-    - kind: ServiceAccount
+    ```bash
+    kubectl -n d8-service-accounts create serviceaccount gitlab-runner-deploy
+    ```
+
+3. Grant the necessary rights to the `ServiceAccount` (using the [ClusterAuthorizationRule](cr.html#clusterauthorizationrule) CR)
+
+    ```bash
```

Comment:
I prefer using `shell` rather than `bash`, as it is more common. But it is not a mistake and is up to you.